crates/bws/src/config.rs

```diff
@@ -36,17 +38,21 @@ impl ProfileKey {
             ProfileKey::server_base => p.server_base = Some(value),
             ProfileKey::server_api => p.server_api = Some(value),
             ProfileKey::server_identity => p.server_identity = Some(value),
+            ProfileKey::state_file_dir => p.state_file_dir = Some(value),
         }
     }
 }
 
-pub(crate) const FILENAME: &str = "config";
-pub(crate) const DIRECTORY: &str = ".bws";
+pub(crate) const BWS_DIRECTORY: &str = ".bws";
+pub(crate) const CONFIG_FILENAME: &str = "config";
```

Comment:
```suggestion
pub(crate) const FILENAME: &str = "config";
pub(crate) const DIRECTORY: &str = ".bws";
```

Reply:
Resolved in [d0c0c10](https://github.com/bitwarden/sdk/pull/388/commits/d0c0c10dadd1a72550a2eccdd861b1a44c8f1859)